eisbuk-admin/src/components/atoms/CustomerList/CustomerGrid.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from "react";
+
+import TextField from "@material-ui/core/TextField";
+
+import makeStyles from "@material-ui/core/styles/makeStyles";
+
+import { Customer } from "eisbuk-shared";
+
+import CustomerGridItem from "./CustomerGridItem";
+
+import CustomerDialog from "./CustomerDialog";
+
+interface Props {
+  customers?: Customer[];
+  className?: string;
+  tableContainerClassName?: string;
+}
+
+const CustomerGrid: React.FC<Props> = ({ customers, className = "" }) => {
+  const classes = useStyles();
+
+  // search flow
+  const [searchString, setSearchString] = useState("");
+  const searchRegex = new RegExp(searchString, "i");
+
+  const [dialogCustomer, setDialogCustomer] = useState<Customer>(
+    {} as Customer
+  );
+
+  const [open, setOpen] = React.useState(false);
```

Comment:
This is obsolete and the `open` prop of `CustomerDialog` can just be `Boolean(dialogCustomer)`